pop/service/contract.go

```diff
@@ -37,149 +37,147 @@ func init() {
 	PoPCoinName = byzcoin.NewInstanceID(h.Sum(nil))
 }
 
-// ContractPopParty represents a pop-party in ByzCoin. It has the following
-// functionalities:
-//   * Spawn - takes a "FinalStatement" argument with the binary representation
-//     of the final statement to store.
-//   * Invoke - hast the following Command
-//     * "Finalize" - that stores a final statement and doesn't let it be
-//       changed afterwards. It will also create a darc for every attendee
-//       and bind that darc to a coin account, putting an initial 1.000.000
-//       popCoins in it.
-//        This command has the following arguments:
-//       * "FinalStatement" - mandatory, to give the new final statement. It
-//         needs to be correctly finalized by the pop-service.
-//       * "Service" - when given, will create a darc and a coin-account for
-//         the service to use.
-func (s *Service) ContractPopParty(cdb byzcoin.ReadOnlyStateTrie, inst byzcoin.Instruction, ctxHash []byte, coins []byzcoin.Coin) (scs []byzcoin.StateChange, cOut []byzcoin.Coin, err error) {
-	cOut = coins
-
-	err = inst.Verify(cdb, ctxHash)
+type contract struct {
+	byzcoin.BasicContract
+	PopPartyInstance
+}
+
+func contractPopParty(in []byte) (byzcoin.Contract, error) {
```

Comment:
the other methods were called `contractPopPartyFromBytes` - I propose to use the same naming.